FRAuth/FRAuth/Config/OAuth2Client.swift

```diff
@@ -207,8 +207,12 @@ public class OAuth2Client: NSObject, Codable {
                     
                     let redirectURL = URL(string: redirectURLAsString)
                     
+                    // Verify that the response state value is the same
+                    if let state = redirectURL?.valueOf("state"), state != pkce.state {
```

Comment:
It looks to me that when the `state` param is omitted by the server the client won't throw an error... 
We should fix that because this is the whole purpose of `state`... reference: [OAuth2 spec](https://datatracker.ietf.org/doc/html/rfc6749#section-4.1.2)

Reply:
Addressed the comment.